scheduler/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+
+networks:
+  scheduler_network:
+    driver: bridge
+
+services:   
+  scheduler:
+    container_name: scheduler
+    build: .
+    restart: always
+    networks:
+      - scheduler_network
```

Comment:
I felt that instead of docker creating a random network when the compose is ran, I should just specify the name of the network. But the network is needed so it can be removed. Let me know what you think